### PR TITLE
cache: add up_get_xcache_linesize() support

### PR DESCRIPTION
**Pull request by GUIDINGLI**
## Summary

cache: add up_get_xcache_linesize() support

## Impact

cache

## Testing

VELA

### CODE REVIEW

File: arch/arm/src/armv7-a/l2cc.h [hunk 1]
```diff
@@ -70,6 +70,22 @@ extern "C"
...
+ *
+ * Input Parameters:
+ *    None
+ *
+ * Returned Value:
+ *    L2 cache linesize
+ *
+ ****************************************************************************/
+
+uint32_t l2cc_get_linesize(void);
```
Comment (pkarashchenko) [inline arch/arm/src/armv7-a/l2cc.h:87]: Maybe this can be a define instead of function?

Comment (GUIDINGLI) [reply arch/arm/src/armv7-a/l2cc.h:87]: l2cc_get_linesize() is a function in l2cc.h, but the MARCO is in l2cc_pl310.h,
l2cc.h is hard to directly include l2cc_pl310.h

Comment (pkarashchenko) [reply arch/arm/src/armv7-a/l2cc.h:87]: Why? Based on the description it is L2CC-P310 specific.

Comment (GUIDINGLI) [reply arch/arm/src/armv7-a/l2cc.h:87]: l2cc_pl310.h is just one type of l2cc.
And l2cc.h is a common header file, l2cc_pl310.h is one of realize of l2cc.

Comment (pkarashchenko) [reply arch/arm/src/armv7-a/l2cc.h:87]: So maybe than let's update the description from `Get L2CC-P310 L2 cache linesize` to `Get L2CC L2 cache linesize`?

Comment (GUIDINGLI) [reply arch/arm/src/armv7-a/l2cc.h:87]: done

**Review by pkarashchenko — APPROVED** [review]
LGTM!

**Pull request merged**